.drone.star

```diff
@@ -321,10 +321,10 @@ def tests(ctx, name, command, phpversions, coverage, xdebugNeeded = False):
     pipelines = []
     if name in config and config[name]:
         for php in phpversions:
-            name = "%s-%s" % (name, php)
+            fullname = "%s-%s" % (name, php)
```

Comment:
We don't want to change the value of `name` - if there are multiple PHP versions, then the previous code resulted in names ending in strings like `-8.1-8.2-8.3`